mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTestWithDefaultTextExtractor.java

```diff
@@ -0,0 +1,39 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.elasticsearch;
+
+    import org.apache.james.mailbox.extractor.TextExtractor;
+    import org.apache.james.mailbox.store.extractor.DefaultTextExtractor;
+    import org.junit.Assume;
+    import org.junit.Test;
+
+public class ElasticSearchIntegrationTestWithDefaultTextExtractor extends AbstractElasticIntegrationTest {
+
+    @Override
+    protected TextExtractor getTextExtractor() {
+        return new DefaultTextExtractor();
+    }
+
+    @Override
+    @Test
+    public void searchWithPDFAttachmentShouldReturnMailsWhenAttachmentContentMatches() throws Exception {
+        Assume.assumeTrue(false);
```

Comment:
I guess there is something better we can do here "@Ignore" did not work

Reply:
generally @Ignore + @Override + empty test body works and is pretty clear about the intent

Reply:
I found something better !